lib/package.json

```diff
@@ -91,5 +92,8 @@
   "sideEffects": [
     "index.js",
     "*.scss"
-  ]
+  ],
+  "resolutions": {
+    "jest-environment-jsdom": "26.6.2"
```

Comment:
`jest-environment-enzyme` still depends on an older version of this that pulls in a `jsdom` version that doesn't have `MutationObserver`.  So we need to override this to use a more current `jsdom`.